cmd/dockle/main.go

```diff
@@ -35,6 +36,15 @@ OPTIONS:
   {{end}}{{$option}}{{end}}{{end}}
 `
 	app := cli.NewApp()
+
+	var dockerSockPath string
+	xdgRuntimeDir := os.Getenv("XDG_RUNTIME_DIR")
+	if xdgRuntimeDir != "" {
+		dockerSockPath = fmt.Sprintf("unix://%s/docker.sock", xdgRuntimeDir)
+	} else {
+		dockerSockPath = "unix:///var/run/docker.sock"
```

Comment:
You can set an empty string if `XDG_RUNTIME_DIR` is not set. So, this line looks unnecessary.

Reply:
yes, but having it in a variable can help to understand what is going to be the default value of the Docker daemon's socket path, WDYT?